packages/core/src/client/utils.ts

```diff
@@ -0,0 +1,14 @@
+export {
+  eq,
+  inArray,
+  or,
+  and,
+  not,
+  gte,
+  lte,
+  gt,
+  lt,
+  asc,
+  desc,
+  getTableColumns,
+} from 'drizzle-orm'
```

Comment:
I opted to put only a subset of tools from drizzle in the console. We always can expand it